uvicorn/config.py

```diff
@@ -172,6 +174,7 @@ def __init__(
         self.ssl_ciphers = ssl_ciphers
         self.headers = headers if headers else []  # type: List[str]
         self.encoded_headers = None  # type: List[Tuple[bytes, bytes]]
+        self.handled_signals = handled_signals
```

Comment:
Let's use `handled_signals or []` here, so that it's *always* a list.
That'll simplify the `install_signal_handlers` implementation slightly.

Reply:
done